### DISCUSSION TITLE
Long axial field of view data

### DISCUSSION THREAD

**Discussion by mr33d** [villekf/OMEGA#36] [General]
Hello,
I was wondering whether OMEGA can reconstruct long axial field of view PET data e.g. LISTMODE data from the Siemens Biograph Quadra in the latest version or if changes to the source code and additional mexw64 files would be required? Many thanks in advance!

**Comment by villekf** [comment]
Hi,

Natively, at the moment no. You can obviously input a sinogram and reconstruct that, but straight list-mode reconstruction is not possible. However, since Quadra should be very similar to Vision (just bigger) it should be possible to modify the Vision code to support Quadra. However, I can't release the source codes without approval from Siemens (and I can't get one at the moment). If I can get proper documentation for Quadra list-mode format, I might have the time to do the modifications myself though. If you have access to the list-mode format documentation and can send it to me, I can at least take a look at it.
